blockchain/processing.py

```diff
@@ -39,17 +39,23 @@
 
 from blockchain.util.crypto import valid_transaction_sig, sign_verification_record, validate_verification_record, sign_subscription, final_hash
 
-from db.postgres import postgres
-from db.postgres import transaction_db
-from db.postgres import verification_db
-from db.postgres import vr_transfers_db
+from bitcoin.core import *
```

Comment:
As mentioned today over the phone, the requirements.txt file was modified to include this library and version. Just clarifying for pull request sake.